appveyor.yml

```diff
@@ -118,6 +121,57 @@ before_build:
 build_script:
   - call ci\appveyor-build.bat
 
+for:
+-
+  matrix:
+    only:
+      - JOB: "R without libarrow"
+  init:
+    ps: |
+          $ErrorActionPreference = "Stop"
+          Invoke-WebRequest http://raw.githubusercontent.com/krlmlr/r-appveyor/master/scripts/appveyor-tool.ps1 -OutFile "..\appveyor-tool.ps1"
+          Import-Module '..\appveyor-tool.ps1'
+
+  before_build:
+    - cd r
+    - ps: Bootstrap
+    - cd %APPVEYOR_BUILD_FOLDER%
+
+  build_script:
+    # TODO: for the version of the R tests that builds, probably can't use this
+    # ci script because we need to build with the Rtools toolchain
+    # - call ci\appveyor-build.bat
+    - pushd r
+    - travis-tool.sh install_deps
+
+  test_script:
+    - travis-tool.sh run_tests
+
+  on_failure:
+    - travis-tool.sh dump_logs
+
+  environment:
+    NOT_CRAN: true
+
+  artifacts:
+    - path: '*.Rcheck\**\*.log'
```

Comment:
Does this works? (See https://ci.appveyor.com/project/ApacheSoftwareFoundation/arrow/builds/25270257/job/06s7p6cktayk9y7w#L914 and https://ci.appveyor.com/project/ApacheSoftwareFoundation/arrow/builds/25270257/job/06s7p6cktayk9y7w#L921)

Reply:
It generally does for R packages, but I'll have to experiment a bit here. I'm guessing the `r/` subdirectory needs to be factored in. 

I'll try that (in the commit I'm about to do to address your comments), but if that doesn't work, I request to defer it to my followup ticket (where I get the libarrow build plugged in) rather than hold this.

Reply:
It's not a blocker, was just curious if this was working.